src/services/SendMailService.ts

```diff
@@ -0,0 +1,51 @@
+import nodemailer, { Transporter } from 'nodemailer';
+import handlebars from 'handlebars';
+import fs from 'fs';
+
+/** @TODO autorização aws */
+class SendMailService {
+  private client: Transporter;
+  constructor() {
+    try {
+      nodemailer.createTestAccount().then((account) => {
```

Comment:
Aqui da pra colocar no inicio do execute, pq é assícrono, ai lá da pra colocar um await antes, pra aguardar terminar, quando for usar o serviço de verdade não é assincrono, ai vai aqui mesmo